test/unstable/safeCast.chpl

```diff
@@ -0,0 +1,22 @@
+var i1 : int =  35;
+var i2 : int(16) = 32;
+var i3 : int(8) = 16;
+var i4 : int = 1;
+i1.safeCast(int);
```

Comment:
Can you wrap `writeln`s around each of these lines so we have some verification of the behavior attached to the deprecation test?